apps/ensindexer/src/lib/plugin-helpers.ts

```diff
@@ -98,12 +94,44 @@ export const activateHandlers =
     handlers,
     ...args
   }: ENSIndexerPluginHandlerArgs<PLUGIN_NAME> & {
-    handlers: Promise<{ default: ENSIndexerPluginHandler<PLUGIN_NAME> }>[];
+    handlers: () => Promise<{ default: ENSIndexerPluginHandler<PLUGIN_NAME> }>[];
```

Comment:
This change enabled lazy evaluation of dynamic imports defined on each plugin, and let resolve errors while running unit test, such as:
![image](https://github.com/user-attachments/assets/7184c942-8f43-4f44-ac70-35a553a719a4)